src/frontend/src/tests/btc/services/btc-send.services.spec.ts

```diff
@@ -0,0 +1,71 @@
+import * as btcSendService from '$btc/services/btc-send.services';
+import { type SendBtcParams } from '$btc/services/btc-send.services';
+import { convertNumberToSatoshis } from '$btc/utils/btc-send.utils';
+import * as backendAPI from '$lib/api/backend.api';
+import * as signerAPI from '$lib/api/signer.api';
+import { ProgressStepsSendBtc } from '$lib/enums/progress-steps';
+import { mapToSignerBitcoinNetwork } from '$lib/utils/network.utils';
+import * as walletUtils from '$lib/utils/wallet.utils';
```

Comment:
Is it possible to not mock the walletUtils or that too much complexity?

Reply:
Actually, it's not required to mock it, my idea was to make sure that it has been called once. If you think it doesn't bring any benefits, I can just remove it, no strong opinion here.

Reply:
Generally speaking, in my opinion, the less the mocks, the better so if not too hard to not mock, yes sure if the test can be asserted without mocking, go for it when it comes to me